website/docs/reference/commands/run.md

```diff
@@ -102,3 +102,96 @@ Encountered an error:
 FailFast Error in model model_1 (models/model_1.sql)
   Failing early due to test failure or runtime error
 ```
+
+## Deferring to previous run state
+
+<Changelog>
+
+ - The `--defer` and `--state` flags are new in dbt v0.18.0
+
+</Changelog>
+
+:::info Beta Feature
+This is net-new functionality in v0.18.0, with iterative improvements to come.
+If you encounter unexpected behavior, please post in Slack or open an issue.
+:::
+
+dbt will resolve `ref` calls differently depending on whether a node is selected
```

Comment:
Might be clearer here to start with an example of a `dbt run` command that includes `--defer` and `--state`. I think it will help frame the first couple of paragraphs for new readers? Even something like "When the --defer flag is provided..." would be welcomed IMO